rollup-dist.config.js

```diff
@@ -0,0 +1,14 @@
+import {
+  createBrowserUmdBuildConfig,
+  createSlimBrowserUmdBuildConfig,
+  createModuleBuild,
+  createTestBuild,
```

Comment:
`createTestBuild ` seems unused now?

Reply:
Only used in `rollup.config.js` (to `/build` dir). I assume this isn't needed in `/dist` as the browser test file is what's needed for your https://benjamine.github.io/jsondiffpatch/test/index.html